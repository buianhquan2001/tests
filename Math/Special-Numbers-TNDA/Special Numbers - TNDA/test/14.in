100
236522711 413026587
214703291 33619910
712421078 846129912
295798597 971348540
592169270 231981436
58547229 535726400
557719071 663171204
836032854 431235172
788040434 141475643
902972682 781268490
581110240 858834376
343119276 820653769
88588043 322187996
869765383 592030992
570507080 67163213
610947061 247300927
822933386 412656435
44020725 16124130
57082611 887682719
874576959 455435084
686070669 486505179
658581161 846544692
411770962 893436409
710904616 36313524
471650787 888579267
179242041 372118734
318460019 321193976
950199089 69236771
296673138 376520976
12342826 866328343
995878028 694914526
681468775 595131298
531935381 127097273
112933177 747484970
189888481 544805843
269708978 176206351
134369351 96775226
788751608 153847582
955952661 795077060
775932509 202493300
170009977 292982199
793907442 158658974
31014608 969136806
195588490 244709756
48470433 461321494
876207154 843513983
185035837 978996126
36293672 67863442
547061700 475515197
85967219 156076960
811137682 935975417
804499293 562288110
961309168 507948716
712914517 730212628
853375152 783158366
863084789 522098573
936820488 754104396
964247906 251718797
173888684 192652952
79339261 962582721
660158480 34522517
818216962 509592130
565455005 613535207
175320927 856654217
393565329 95119070
717886390 273269717
148748438 944417340
40969115 521108558
568038430 513141964
272894261 354853562
47892030 207526688
411794220 915091165
733190407 515292110
995122147 224428535
55607577 327612447
777514095 747767710
7267520 712655246
604686222 906792782
775946185 378641246
206638431 338937141
476587504 542924558
595262918 829061905
893324712 261621978
829528608 9713935
34732922 166061964
122037349 233141549
109796750 620124261
436226330 650674652
465041906 722883487
71148031 749493103
354436666 859390959
680354365 284260627
699477192 423010436
464525766 449803154
796508821 327393649
506349803 66876106
244545381 408839174
111793595 540192212
328135983 170748287
232656959 105989803
