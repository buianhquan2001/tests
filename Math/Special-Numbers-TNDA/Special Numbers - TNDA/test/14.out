354379146
42369660
702636235
310671536
435518689
309577859
637753609
57312125
624731838
453770677
328718641
731084767
83272590
334856451
774551556
124975058
10700548
602456326
528030127
30090196
424059235
608589010
807179511
612003689
801414673
563469969
196761795
767844956
693103273
853784329
433992303
395805970
765562138
799297248
678703009
997740845
563272093
253347990
699848312
621806000
779960289
557962978
695974410
811458638
173517685
885596310
983095747
945802715
705738850
669224735
868581219
602520955
242831678
734315860
279415957
150929049
84814744
401150063
218073729
715730218
143144599
144105081
321767461
801111052
778130277
356525632
923510893
36085949
892361316
168297818
14288831
971522249
455944437
541426261
639008555
37255833
647894292
298422860
269324371
947397635
442386362
546315867
41711383
767753945
116327378
113898636
100653377
643115538
673749073
540007631
193456077
190280351
803024605
583537595
832355900
761435567
29212534
597871853
7564660
820823576
